author/topics/document-format/text.adoc

```diff
@@ -116,25 +128,38 @@ Table 1 in the AsciiISO Rice example document illustrates
 a large range of table formatting options.
 ====
 
-=== Count of table header and footer rows
+== Mathematical expressions
 
-In Asciidoc, a table can have at most one header row or footer row. In Metanorma,
-a nominal single header row is routinely broken up into multiple rows in order
-to accommodate units or symbols, that line up against each other, though
-they are displayed as merged cells with no grid between them. To address this,
-tables can be marked up with an optional `headerrows` attribute:
+Like Asciidoctor proper, Metanorma Asciidoctor accepts mathematical input
+in either LaTeX or AsciiMath, following the conventions of Asciidoctor:
 
-[source,asciidoctor]
---
-[headerrows=2]
-|===
-.2+|Defect 4+^| Maximum permissible mass fraction of defects in husked rice +
-stem:[w_max]
-| in husked rice | in milled rice (non-glutinous) | in husked parboiled rice | in milled parboiled rice
+* The document attribute `:stem:` means any markup tagged as `[stem]`
+(`stem:[...]`, or `[stem]` before a block delimited with `++++`)
```

Comment:
Going to replace `++++` with `pass:[++++]`, otherwise this renders as nothing.

Reply:
OK